tests/Unit/WP_Mock/HookTest.php

```diff
@@ -54,6 +54,7 @@ public function callback(): bool
         yield 'closure (object)' => [$closureInstance, '__CLOSURE__'];
         yield 'closure (representation)' => ['<Closure>', '__CLOSURE__'];
         yield 'closure (class name)' => [Closure::class, '__CLOSURE__'];
+        yield 'mockery matcher' => [\Mockery::type(\Closure::class), '__CLOSURE__'];
```

Comment:
```suggestion
        yield 'closure (Mockery matcher)' => [Mockery::type(Closure::class), '__CLOSURE__'];
```

can avoid using fully qualified namespaces (add `Mockery` namespace reference at the top of the class)